pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "securesystemslib"
+authors = [{name = "https://www.updateframework.com", email = "theupdateframework@googlegroups.com"}]
+license = {text = "MIT"}
+description = "A library that provides cryptographic and general-purpose routines for Secure Systems Lab projects at NYU"
+readme = "README.rst"
+keywords = [
+    "cryptography",
+    "keys",
+    "signatures",
+    "rsa",
+    "ed25519",
+    "ecdsa",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: POSIX",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Security",
+    "Topic :: Software Development",
+]
+requires-python = "~=3.7"
+dependencies = [
+    "six>=1.11.0",
+    'subprocess32; python_version < "3"',
+    "python-dateutil>=2.8.0",
+]
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://github.com/secure-systems-lab/securesystemslib"
+Source = "https://github.com/secure-systems-lab/securesystemslib"
+Issues = "https://github.com/secure-systems-lab/securesystemslib/issues"
+
+[project.optional-dependencies]
+crypto = ["cryptography>=37.0.0"]
+pynacl = ["pynacl>1.2.0"]
+testing = ['mock; python_version < "3.3"']
+PySPX = ["PySPX==0.5.0"]
+
+[tool.setuptools]
+include-package-data = false
```

Comment:
How does this relate to MANIFEST.in? I think we want to include some non-*.py files in source distributions.